aws/deployment-role-policy.json

```diff
@@ -0,0 +1,145 @@
+          "Resource": "arn:aws:iam::*:role/*"
+      },
+      {
+          "Sid": "LambdaFunctionPermissions",
+          "Action": [
+              "lambda:CreateFunction",
+              "lambda:UpdateFunctionCode",
+              "lambda:CreateAlias",
+              "lambda:InvokeFunction",
+              "lambda:GetLayerVersion",
+              "lambda:GetEventSourceMapping",
+              "lambda:TagResource",
+              "lambda:GetFunction",
+              "lambda:PublishVersion",
+              "lambda:PublishLayerVersion",
+              "lambda:ListVersionsByFunction",
+              "lambda:GetFunctionConfiguration",
+              "lambda:UpdateFunctionConfiguration",
+              "lambda:CreateEventSourceMapping",
+              "lambda:UpdateEventSourceMapping",
+              "lambda:AddPermission",
+              "lambda:DeleteLayerVersion",
+              "lambda:DeleteFunction",
+              "lambda:DeleteAlias",
+              "lambda:DeleteEventSourceMapping",
+              "lambda:RemovePermission"
+          ],
+          "Effect": "Allow",
+          "Resource": "*"
+      },
+      {
+          "Sid": "EC2Permissions",
+          "Action": [
+              "ec2:DescribeVpcs",
+              "ec2:DescribeSubnets",
+              "ec2:DescribeSecurityGroups"
+          ],
+          "Effect": "Allow",
+          "Resource": "*"
+      },
+      {
+          "Sid": "S3BucketPermissions",
+          "Action": [
+              "s3:DeleteBucket"
+          ],
+          "Effect": "Allow",
+          "Resource": "arn:aws:s3:::*"
+      },
+      {
+          "Sid": "S3AccessPointPermissions",
+          "Action": [
+              "s3:CreateAccessPoint",
+              "s3:DeleteAccessPoint",
+              "s3:GetAccessPoint",
+              "s3:GetAccessPointPolicy",
+              "s3:CreateAccessPointForObjectLambda",
+              "s3:DeleteAccessPointForObjectLambda",
+              "s3:GetAccessPointForObjectLambda",
+              "s3:GetAccessPointConfigurationForObjectLambda",
+              "s3:GetAccessPointPolicyStatusForObjectLambda"
+          ],
+          "Effect": "Allow",
+          "Resource": [
+              "arn:aws:s3:*:*:accesspoint/*",
+              "arn:aws:s3-object-lambda:*:*:accesspoint/*"
+          ]
+      },
+      {
+          "Sid": "S3ObjectPermissions",
+          "Action": [
+              "s3:GetObject"
+          ],
+          "Effect": "Allow",
+          "Resource": "arn:aws:s3:::*/*"
+      },
+      {
+          "Sid": "SNSTopicPermissions",
+          "Action": [
+              "sns:TagResource",
+              "sns:GetTopicAttributes",
+              "sns:DeleteTopic",
+              "sns:CreateTopic",
+              "sns:SetTopicAttributes",
+              "sns:Subscribe",
+              "sns:Unsubscribe",
+              "sns:ListSubscriptionsByTopic"
+          ],
+          "Effect": "Allow",
+          "Resource": "arn:aws:sns:*:*:*"
+      }
+  ]
```

Comment:
Would you please make this block's indent same as others? So that there's won't be lots of space diffs when we update this in the future.
And adding a change line character in the end of the file would be helpful for future diff as well.

Reply:
Sorry, @jack-c-tang I don't understand what you mean by that, could you elaborate?

Reply:
Please make sure all indents have same space characters.

Reply:
Fixed!